aicoe/sesheta/utils.py

```diff
@@ -88,26 +89,35 @@
     "Dominik Tuchyna": "101087488035276666197",
     "Francesco Murdaca": "108929048208403662680",
     "Frido Pokorny": "106810069271823707995",
+    "Gage Krumbach": "113871793657874355068",
```

Comment:
The `GITHUB_REALNAME_MAP` should also be updated with the new additions:

https://github.com/AICoE/sefkhet-abwy/blob/653be9740a2650d22c34e41c85ab56944a66ed48/aicoe/sesheta/utils.py#L55-L56

Note that github usernames should be all lowercase there (see [here](https://github.com/AICoE/sefkhet-abwy/blob/653be9740a2650d22c34e41c85ab56944a66ed48/aicoe/sesheta/utils.py#L166)), so please take the opportunity to fix @Gregory-Pereira's entry by converting to lowercase

Reply:
uhhhh, nice catch!